manager-dashboard/app/components/TutorialList/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+import {
+    getDatabase,
+    ref,
+    query,
+    orderByChild,
+    equalTo,
+} from 'firebase/database';
+import { BsJournalBookmarkFill } from 'react-icons/bs';
+
+import useFirebaseDatabase from '#hooks/useFirebaseDatabase';
+import usePagination from '#hooks/usePagination';
+import PendingMessage from '#components/PendingMessage';
+import Pager from '#components/Pager';
+
+import styles from './styles.css';
+
+// FIXME: these are common types
+export type ProjectType = 1 | 2 | 3 | 4;
+const projectTypeLabelMap: {
+    [key in ProjectType]: string
+} = {
+    1: 'Build Area',
+    2: 'Footprint',
+    3: 'Change Detection',
+    4: 'Completeness',
+};
+
```

Comment:
yes. I would encourage to define this somewhere else as these might get used in other parts of the manager dashboard as well.